RELEASE_NOTES.md

```diff
@@ -4,12 +4,23 @@ Release Notes
 v1.0.0-beta.x.x
 ---------------
 
+_This release breaks binary compatibility and may break source
+compatibility with existing codebases, for both hosts and managers. See
+breaking changes section for more details._
+
 ### Breaking changes
 
 - Removed official support for Python 3.7. The minimum supported Python
   version is now Python 3.9.
   [#1365](https://github.com/OpenAssetIO/OpenAssetIO/pull/1365)
 
+- Switched to `std::set` for trait sets (via `TraitSet` alias) in C++.
```

Comment:
Doesn't say what you switched from in the first sentence. Should it?

unordered_set is mentioned further down.

More of a question than a problem.

Reply:
The first thing that popped into my head when reading "Switched to 'std::set'" was... switched from what?

So personally I think it's useful.

Reply:
OK agreed. Updated in 9db400fc